tutor/commands/compose.py

```diff
@@ -84,10 +84,7 @@ def job_runner(self, config: Config) -> ComposeTaskRunner:
 @click.option("--skip-build", is_flag=True, help="Skip building Docker images")
 @click.pass_context
 def launch(
-    context: click.Context,
-    non_interactive: bool,
-    pullimages: bool,
-    skip_build: bool,
+    context: click.Context, non_interactive: bool, pullimages: bool, skip_build: bool
```

Comment:
this change is opposite to the change you made on line 187. It would be good if we can maintain consistency where possible.

Reply:
Nice catch.